src/jit/codegenarm64.cpp

```diff
@@ -2616,9 +2616,15 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                 if (targetType == TYP_STRUCT)
                 {
                     // At this point any TYP_STRUCT LclVar must be a two register argument
-                    assert(varDsc->lvSize() == 16); 
-                    emit->emitIns_R_S(ins_Load(TYP_I_IMPL), emitTypeSize(TYP_I_IMPL), targetReg, varNum, 0);
-                    emit->emitIns_R_S(ins_Load(TYP_I_IMPL), emitTypeSize(TYP_I_IMPL), REG_NEXT(targetReg), varNum, TARGET_POINTER_SIZE);
+                    assert(varDsc->lvSize() == 2*TARGET_POINTER_SIZE);
```

Comment:
I agree we can all discuss the proper approaches for LSRA when Carol is in town in Feburary.